spring-kafka-3/src/main/java/com/baeldung/spring/kafka/start/stop/consumer/Constants.java

```diff
@@ -0,0 +1,6 @@
+package com.baeldung.spring.kafka.start.stop.consumer;
```

Comment:
Couldn't we put all these in a package called "startstopconsumer" like you did for the live test?

Reply:
I have moved the live test package from "startstopconsumer" to "start.stop.consumer", so it is now consistent.